src/main/java/org/gridsuite/loadflow/server/entities/parameters/LoadFlowParametersEntity.java

```diff
@@ -0,0 +1,209 @@
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import org.gridsuite.loadflow.server.dto.parameters.LoadFlowParametersInfos;
+import org.gridsuite.loadflow.server.dto.parameters.LoadFlowParametersValues;
+
+/**
+ * @author Ayoub LABIDI <ayoub.labidi at rte-france.com>
+ */
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Entity
+@Table(name = "loadFlowParameters")
+public class LoadFlowParametersEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private UUID id;
+
+    @Column(name = "voltageInitMode")
+    @Enumerated(EnumType.STRING)
+    private LoadFlowParameters.VoltageInitMode voltageInitMode;
+
+    @Column(name = "transformerVoltageControlOn", columnDefinition = "boolean default false")
+    private boolean transformerVoltageControlOn;
+
+    @Column(name = "useReactiveLimits", columnDefinition = "boolean default true")
+    private boolean useReactiveLimits;
+
+    @Column(name = "phaseShifterRegulationOn", columnDefinition = "boolean default false")
+    private boolean phaseShifterRegulationOn;
+
+    @Column(name = "twtSplitShuntAdmittance", columnDefinition = "boolean default false")
+    private boolean twtSplitShuntAdmittance;
+
+    @Column(name = "shuntCompensatorVoltageControlOn", columnDefinition = "boolean default false")
+    private boolean shuntCompensatorVoltageControlOn;
+
+    @Column(name = "readSlackBus", columnDefinition = "boolean default true")
+    private boolean readSlackBus;
+
+    @Column(name = "writeSlackBus", columnDefinition = "boolean default false")
+    private boolean writeSlackBus;
+
+    @Column(name = "dc", columnDefinition = "boolean default false")
+    private boolean dc;
+
+    @Column(name = "distributedSlack", columnDefinition = "boolean default true")
+    private boolean distributedSlack;
+
+    @Column(name = "balanceType")
+    @Enumerated(EnumType.STRING)
+    private LoadFlowParameters.BalanceType balanceType;
+
+    @Column(name = "dcUseTransformerRatio", columnDefinition = "boolean default true")
+    private boolean dcUseTransformerRatio;
+
+    @Column(name = "countriesToBalance")
+    @ElementCollection
+    @CollectionTable(foreignKey = @ForeignKey(name = "loadFlowParametersEntity_countriesToBalance_fk1"),
+            indexes = {@Index(name = "loadFlowParametersEntity_countriesToBalance_idx1",
+                    columnList = "load_flow_parameters_entity_id")})
+    private Set<String> countriesToBalance;
+
+    @Column(name = "connectedComponentMode")
+    @Enumerated(EnumType.STRING)
+    private LoadFlowParameters.ConnectedComponentMode connectedComponentMode;
+
+    @Column(name = "hvdcAcEmulation", columnDefinition = "boolean default true")
+    private boolean hvdcAcEmulation;
+
+    @Column(name = "dcPowerFactor", columnDefinition = "double default 1.0")
+    private double dcPowerFactor;
```

Comment:
You need to keep in sync database default values with Java class initial values, as you're using primitive type so they always have a default value, never `null`.
```suggestion
    @Column(name = "transformerVoltageControlOn", columnDefinition = "boolean default false")
    private boolean transformerVoltageControlOn = false;

    @Column(name = "useReactiveLimits", columnDefinition = "boolean default true")
    private boolean useReactiveLimits = true;

    @Column(name = "phaseShifterRegulationOn", columnDefinition = "boolean default false")
    private boolean phaseShifterRegulationOn = false;

    @Column(name = "twtSplitShuntAdmittance", columnDefinition = "boolean default false")
    private boolean twtSplitShuntAdmittance = false;

    @Column(name = "shuntCompensatorVoltageControlOn", columnDefinition = "boolean default false")
    private boolean shuntCompensatorVoltageControlOn = false;

    @Column(name = "readSlackBus", columnDefinition = "boolean default true")
    private boolean readSlackBus = true;

    @Column(name = "writeSlackBus", columnDefinition = "boolean default false")
    private boolean writeSlackBus = false;

    @Column(name = "dc", columnDefinition = "boolean default false")
    private boolean dc = false;

    @Column(name = "distributedSlack", columnDefinition = "boolean default true")
    private boolean distributedSlack = true;

    @Column(name = "balanceType")
    @Enumerated(EnumType.STRING)
    private LoadFlowParameters.BalanceType balanceType;

    @Column(name = "dcUseTransformerRatio", columnDefinition = "boolean default true")
    private boolean dcUseTransformerRatio = true;

    @Column(name = "countriesToBalance")
    @ElementCollection
    @CollectionTable(foreignKey = @ForeignKey(name = "loadFlowParametersEntity_countriesToBalance_fk1"),
            indexes = {@Index(name = "loadFlowParametersEntity_countriesToBalance_idx1",
                    columnList = "load_flow_parameters_entity_id")})
    private Set<String> countriesToBalance;

    @Column(name = "connectedComponentMode")
    @Enumerated(EnumType.STRING)
    private LoadFlowParameters.ConnectedComponentMode connectedComponentMode;

    @Column(name = "hvdcAcEmulation", columnDefinition = "boolean default true")
    private boolean hvdcAcEmulation = true;

    @Column(name = "dcPowerFactor", columnDefinition = "double default 1.0")
    private double dcPowerFactor = 1.0;
```